README.md

```diff
@@ -1,70 +1,198 @@
-# Getting Started with Create React App
+# 📗 Table of Contents
```

Comment:
- [ ] Your frontend project contains way many linter issues for me to be able to list here.   Before resubmitting for review, kindly run the command

```
npx eslint --fix .
```
 
so they autofix, commit and push.

That way, your code will comply with quality standards. 💪

Reply:
Thank you so much for taking the time to review our code. We will make corrections and resubmit. as for the linters, I ran `npx eslint --fix . ` before submitting, I don't know why you are still getting the errors.